### PR TITLE
DTSPB-778 Case Progress Tab

### PR DESCRIPTION
**Pull request by vinceholt**
### JIRA links ###
https://tools.hmcts.net/jira/browse/DTSPB-715
https://tools.hmcts.net/jira/browse/DTSPB-776
https://tools.hmcts.net/jira/browse/DTSPB-777
https://tools.hmcts.net/jira/browse/DTSPB-778
https://tools.hmcts.net/jira/browse/DTSPB-779
https://tools.hmcts.net/jira/browse/DTSPB-780
https://tools.hmcts.net/jira/browse/DTSPB-781
https://tools.hmcts.net/jira/browse/DTSPB-784

### Change description ###
Add Case Progress Tab for Professional Users. to show custom html for:
1) Stopped State
2) Escalated to Registrar State
3) For any other state, a Task List, showing which sections of the application are complete, incomplete or not started

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
+{
```
Comment (meganaslattery) [inline .vscode/launch.json:1]: Not sure we want a vscode file added to the code base?

**Review by meganaslattery — APPROVED** [review]
It will have to go in after DTSPB-922 but it looks good!

File: .vscode/launch.json (hunk 1 above)
Comment (vinceholt) [reply .vscode/launch.json:1]: ah yes went through this with Jack, the jury is out on this on the internet, with camps for and against; but the aginst camp is the one where lots of individuals have their own launch.json entries for their own purposes, we don't have that, and mor often than not if you want to run something from package.json in the debugger, you'd want others to be able to do that too - e.g. mocha test an individual file. So I think in our context, just invoking package.json entries, it's beneficial for others to be able to inherit shared invocations to save having to manually type in a terminal window, and we're unlikely to have entries other than package.json invoke. Doesn't harm or affect user of other IDEs

**Comment by vinceholt** [comment]
replaced with a new pr for the epic

**Pull request closed without merging**